oak_tensorflow_bin/layout.ld

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
What's the plan to keep this in sync with the other binaries @andrisaar @conradgrobler  ?

Reply:
We don't yet have a good plan for the linker scripts. I vaguely remember that one linker script can reference another, so we might be able to move a lot of it to a central script.

We should reuse the asm between the different binaries and do general deduplication where possible before creating any new binaries that are based on the restricted kernel.

Reply:
Perhaps we could use some symlinks for the time being, instead of maintaining several copies in our repo?